streams/src/main/java/org/apache/kafka/streams/kstream/TransformerSupplier.java

```diff
@@ -33,7 +33,6 @@
  * @param <R> {@link org.apache.kafka.streams.KeyValue KeyValue} return type (both key and value type can be set
  *            arbitrarily)
  * @see Transformer
- * @see KStream#transform(TransformerSupplier, String...)
```

Comment:
As for `Transformer`

Reply:
Although we remove it, I think it helpful to update it to prevent `gradle javadoc` to fail. I am checking if the modifications are right this way. Is there any problem if I go this way? Or some suggestion?